distribution/lib/Standard/Base/0.0.0-dev/src/Runtime/Thread.enso

```diff
@@ -19,3 +20,18 @@ import project.Any.Any
          Thread.with_interrupt_handler (1 + 1) <| IO.println "I died!"
 with_interrupt_handler : Any -> Any -> Any
 with_interrupt_handler ~action ~interrupt_handler = @Builtin_Method "Thread.with_interrupt_handler"
+
+## PRIVATE
+   ADVANCED
+   Temporarily cease execution of the current thread.
+
+   Arguments:
+   - time: amount of milliseconds to sleep
+
+   > Example
+     Sleep the current thread for 1 second.
+
+         Thread.sleep 1000 <| IO.println "I continue!"
+sleep : Integer
+sleep time_in_milliseconds = @Builtin_Method "Thread.sleep"
```

Comment:
Nice to get this as a builtin, I was always `polyglot java import java.lang.Thread`, but this seems so much nicer

Reply:
I'd rather limit builtins to a bare minimum. Especially something that doesn't have to be fast as `sleep` doesn't indicate any need to be a builtin.